From 0a0fd5af99f2ae9b0f8cc6b943b98b7be43ed723 Mon Sep 17 00:00:00 2001
From: Michael Natterer <mitch@gimp.org>
Date: Wed, 02 Nov 2011 19:27:39 +0000
Subject: Bug 662633 - Scheduled transaction editor crashes with gtk+-2.24.7

Fix commit a516d2359c9eac84bfa4682a70a62315adedb1d6: check if
priv->arrow_button exists in forall().
---
diff --git a/gtk/gtktoolbar.c b/gtk/gtktoolbar.c
index edeab88..b2c4b15 100644
--- a/gtk/gtktoolbar.c
+++ b/gtk/gtktoolbar.c
@@ -2532,7 +2532,7 @@ gtk_toolbar_forall (GtkContainer *container,
       list = next;
     }
   
-  if (include_internals)
+  if (include_internals && priv->arrow_button)
     callback (priv->arrow_button, callback_data);
 }
 
--
cgit v0.9.0.2
